src/tools/quemap/monitor.c

```diff
@@ -259,4 +277,5 @@ void Mon_Shutdown(const char *msg) {
 	mon_backlog = NULL;
 
 	Net_Shutdown();
+#endif
```

Comment:
Oh, I wasn't sure if it was necessary or just an external cool thingy. I'll see about getting libxml included in a PR.